### ISSUE TITLE
Accessibility color contrast

### ISSUE THREAD

**Issue by Arhell** [dotnet-foundation/website#887]
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Target application: Community | .NET Foundation - https://dotnetfoundation.org/community

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.57 (foreground color: #006cff, background color: #ebe0eb, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
